templates/import_file_process.tt.js

```diff
@@ -0,0 +1,32 @@
+
+var poll_n = 0;
+var timeout = 5000;
+var job_info_state;
+
+var statuses = {
+	"inactive" : "$Lang{job_status_inactive}{$lc}",
```

Comment:
This syntax only works inside Perl code, it will not work inside the template.
It needs to be replaced by [% lang("job_status_inactive") %]